src/main/resources/application.yaml

```diff
@@ -67,6 +67,11 @@ hearingNoticeDocument.fileExtension: PDF
 hearingNoticeDocument.fileName: "hearing-notice"
 hearingNoticeDocument.templateName: ${IA_HEARING_NOTICE_TEMPLATE:TB-IAC-HNO-ENG-00241.docx}
 
+remoteHearingNoticeDocument.contentType: application/pdf
+remoteHearingNoticeDocument.fileExtension: PDF
+remoteHearingNoticeDocument.fileName: "hearing-notice"
+remoteHearingNoticeDocument.templateName: ${IA_REMOTE_HEARING_NOTICE_TEMPLATE:Remote_hearing_notice.docx}
```

Comment:
This template name will need to change as soon as uploaded to SharePoint.

Reply:
Template name updated.